### ISSUE TITLE
pytest_check is the only pytest plugin that uses an underscore in its name

### ISSUE THREAD

**Issue by jeffwright13** [okken/pytest-check#83]
Apparently, this is the only pytest plugin with an underscore in its name, rather than a dash. I searched [here](https://docs.pytest.org/en/7.0.x/reference/plugin_list.html) for `pytest_` and found no other modules named this way. I don't know if this has any impact on anything at all (likely not).

When I do a local list of installed modules, again, this one is the only one with an underscore.
```
pytest                6.2.5
pytest-bdd            5.0.0
pytest_check          1.0.5
pytest-cov            2.7.1
pytest-forked         1.4.0
pytest-html           3.1.1
pytest-metadata       2.0.1
pytest-rerunfailures  10.2
pytest-runner         4.5.1
pytest-timeout        2.1.0
pytest-tui            0.9.4
pytest-xdist          2.5.0
```
Not worried about it, but thought I would bring it to your attention.

**Comment by okken** [comment]
decent point.

**Comment by jeffwright13** [comment]
I ran into this same issue with [one of my earlier repos](https://pypi.org/project/audio-program-generator/). I'm trying to remember now if there were any ramifications. I seem to recall that part of the build process changes underscsores into dashes, and that led to confusion (and possibly build issues) - I'll have to go research it.

**Comment by okken** [comment]
fixed with version 1.0.6
